gossip/client.go

```diff
@@ -41,7 +41,6 @@ type client struct {
 	rpcClient   *rpc.Client   // RPC client
 	forwardAddr net.Addr      // Set if disconnected with an alternate addr
 	lastFresh   int64         // Last wall time client received fresh info
-	err         error         // Set if client experienced an error
```

Comment:
It's a weird error handling pattern and wasn't actually useful in practice; Right now the error is logged as soon as a client is released, which I think is much more useful for anyone trying to match the error to a line in the source.